pkg/controller/volume/persistentvolume/pv_controller.go

```diff
@@ -1326,7 +1330,9 @@ func (ctrl *PersistentVolumeController) provisionClaimOperation(claimObj interfa
 		return
 	}
 
+	opComplete := util.OperationCompleteHook(plugin.GetPluginName(), "volume_provision")
```

Comment:
curious too, but regardless of answer we decided to wrap Provision since wrapping provisionClaimOperation would include irrelevant API server delays